packages/runtime/runtime-definitions/src/id-compressor/persisted-types/0.0.1.ts

```diff
@@ -8,7 +8,7 @@ import type {
 	LocalCompressedId,
 	OpSpaceCompressedId,
 	SessionId,
-} from ".././identifiers";
+} from "../identifiers";
```

Comment:
Hijacking this line (I can't comment on unchanged lines) to note the readme for 0.0.1.ts is totally wrong. It mentions SharedTree and some other stuff.